app/controllers/concerns/state_file/repeated_question_concern.rb

```diff
@@ -21,6 +22,8 @@ def prev_path
     end
 
     def next_path
+      return super if params[:return_to_review].present?
+
```

Comment:
this works but I still need to walk myself through this to be 100% comfortable with it 😬

Reply:
concern 1: idk if repeated question concern _always_ needs to include `ReturnToReviewConcern`.